### PR TITLE
WIP: implements emmet monaco

### PR DESCRIPTION
**Pull request by thekiba**
🦄emmet + monaco = 💕

### CODE REVIEW

**Comment by thekiba** [comment]
Wating when troy351/emmet-monaco-es#3 will resolved.

File: libs/code-demos/src/lib/code-demo-editor/directives/code-demo-editor.emmet.directive.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    const isCssActivated = isActivatedLanguage('css', event);
+    if (isCssActivated) {
+      console.warn('Css are not supported css yet!');
+      this.activateEmmetWith('css');
+    }
+  }
+
+  private activateEmmetWith(language: 'html' | 'css') {
+    const editor = this.editorInjector.editor;
+    if (language === 'html') {
```
Comment (NothingEverHappens) [inline libs/code-demos/src/lib/code-demo-editor/directives/code-demo-editor.emmet.directive.ts]: String enum pl

Comment (thekiba) [reply libs/code-demos/src/lib/code-demo-editor/directives/code-demo-editor.emmet.directive.ts]: 🗽

**Comment by NothingEverHappens** [comment]
Is this ecehappening?
Or should I close?

**Comment by NothingEverHappens** [comment]
Does not seem like this is going to happen? Closing, feel free to reopen

**Pull request closed without merging**